.github/workflows/verify.yaml

```diff
@@ -1,4 +1,4 @@
-name: verify
+name: Verify
```

Comment:
In general, I am not too much taking care of how somebody names something as long as it has no consequences. Here we had a repository with consistent naming (everything was lower case). And changes to those names require changes to the branch protection configuration. So, assuming you just changed this because of personal preferences, I suggest to not do those changes.

Reply:
I would usually agree with that, but my intention is to bring all repositories into a state of similarity with regards to the naming of actions that are performed, as the previous list of actions/tests that are run were not very descriptive and as we add more it would make sense to ensure that they are categorized for ease of reading.

![image](https://github.com/shipwright-io/triggers/assets/5060301/4da6790f-453b-4ba0-a23a-effb98f40ae2)